### DISCUSSION TITLE
Uninstaller issues with Adobe AIR

### DISCUSSION THREAD

**Discussion by SeagoMac** [airsdk/Adobe-Runtime-Support#3408] [General]
I hope you guys cab help me. I am trying to uninstall AIR runtime - version 51.1.1.3 on a Mac running Sonoma 14.6.1 so that I can install AIR runtime including WebKit - version 33.1.1.744. I need this to run Juicebox Builder Pro.

I have run the Adobe AIR Uninstaller.app in the utilities folder and run it through the terminal using the sudo command -
% sudo /Applications/Utilities/Adobe\ AIR\ Uninstaller.app/Contents/MacOS/Adobe\ AIR\ Installer

But every time I try and run the AIR runtime including WebKit - version 33.1.1.744 it says
<img width="481" alt="Screenshot 2024-08-16 at 18 39 30" src="https://github.com/user-attachments/assets/b29ce893-4bda-4aeb-a09a-2d35a5d29ae3">
It can't install because there is a newer version installed.

Any ideas as to how I can get around this problem it would be greatly appreciated .

**Comment by ajwfrost** [comment]
Hi

It sounds like you may have got a left-over element from an Adobe version of the AIR runtime..

> Adobe AIR Uninstaller.app in the utilities folder

We don't put an 'uninstaller' into the Applications/Utilities folder. To uninstall the runtime, you can just launch the "/Applications/Adobe AIR Installer" application. Launching this without arguments will put it into 'uninstaller' mode.

**Reply by SeagoMac** [reply]
Many thanks for your information, that worked. I updated to the latest version of Adobe air and then ran the installer that was in the applications folder and not the uninstaller that was in the Utilities folder and uninstalled Adobe AIR. That allowed me to then install the AIR runtime including WebKit - version 33.1.1.744.
